x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph/layout_graph.ts

```diff
@@ -71,6 +78,27 @@ export const layoutGraph = (
       res.children.forEach((child) => {
         nodesById[child.data.id] = child;
       });
+    } else {
+      // Any other shape i.e. hexagon, diamond, ...
+      const hasLabelOrId = !!node.data.label || !!node.data.id;
+      const hasIps = node.data.ips && node.data.ips.length > 0;
+      const hasCountryCodes = node.data.countryCodes && node.data.countryCodes.length > 0;
+
+      size.height += NODE_TAG_HEIGHT;
+
+      if (hasLabelOrId) {
+        size.height += NODE_LABEL_HEIGHT;
+      }
+      if (hasLabelOrId || hasIps || hasCountryCodes) {
+        size.height += NODE_DETAILS_GAP;
+      }
+      if (hasIps) {
+        size.height += NODE_IPS_HEIGHT;
+      }
+      if (hasCountryCodes) {
+        // TODO What if countryCodes contains invalid ones and actual array is empty?
+        size.height += NODE_COUNTRY_FLAGS_HEIGHT;
```

Comment:
Eventually you can end up with height that is not multiplication of 20.. 

I'd suggest to use cieling to the next 20 multiplication